### ISSUE TITLE
Dark Theme needs to be selective by user

### ISSUE THREAD

**Issue by jrlandis525** [RocketChat/Rocket.Chat#24144]
## My Setup
- Operating System: Windows 10
- App Version: 2.14.7
- Installation type: 

- [x] I have tested with the latest version 
- [x] I can simulate the issue easily

## Description
 The dark theme is currently a global selection.  While it is good for most users, there are still a few in my company that would rather not use it as it is a bit harder on our eyes

### Expected Behavior
  There should be a choice by the user whether they would like to use the light theme or dark theme.

**Comment by dougfabris** [comment]
Hi there, thanks for the contribution! 🚀 💯 

During this year we've been working on a bunch of accessibility issues and I'm glad to say we do have the dark theme available and even a high-contrast theme. It can be accessible on User Menu > Accessibility and appearance!

---

**Questions? Help needed? Feature Requests?** 

- Join our [Open Server](https://open.rocket.chat/) in the #support channel and feel free to raise a question
- Join our [Community Forum](https://forums.rocket.chat/) and search/create a post there
